test/commands/force/data/soql/query/dataSoqlQuery.nut.ts

```diff
@@ -116,6 +116,16 @@ describe('data:soql:query command', () => {
       expect(stdError).to.include('unexpected token');
     });
 
+    it('should produce correct error when invalid soql provided', () => {
+      const filepath = path.join(testSession.dir, 'soql.txt');
```

Comment:
the `MALFORMED_QUERY` error is returned by the server, we already have a NUT that tests this scenario (the one above this).

Since there's already a NUT that covers this soqlqueryfile flag does it makes sense to keep this one?